### PR TITLE
Add more packets

### PR DESCRIPTION
**Pull request by basaigh**

### CODE REVIEW

File: java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/Identifier.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.cloudburstmc.protocol.java.data;
+
+import lombok.Value;
+
+@Value
+public class Identifier {
```
Comment (Redned235) [inline java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/Identifier.java]: Use Kyori's `Key` instead.

File: java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/command/properties/EntityProperties.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package org.cloudburstmc.protocol.java.data.command.properties;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@Value
+@AllArgsConstructor
+public class EntityProperties implements CommandProperties{
```
Comment (Redned235) [inline java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/command/properties/EntityProperties.java]:
```suggestion
public class EntityProperties implements CommandProperties {
```

File: java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/command/properties/FloatProperties.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package org.cloudburstmc.protocol.java.data.command.properties;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@Value
+@AllArgsConstructor
+public class FloatProperties implements CommandProperties{
```
Comment (Redned235) [inline java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/command/properties/FloatProperties.java]:
```suggestion
public class FloatProperties implements CommandProperties {
```

File: java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/command/properties/IntegerProperties.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package org.cloudburstmc.protocol.java.data.command.properties;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@Value
+@AllArgsConstructor
+public class IntegerProperties implements CommandProperties{
```
Comment (Redned235) [inline java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/command/properties/IntegerProperties.java]:
```suggestion
public class IntegerProperties implements CommandProperties {
```

**Review by Redned235 — CHANGES_REQUESTED** [review]

File: java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/Identifier.java (hunk 1 above)
Comment (basaigh) [reply java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/Identifier.java]: I think I should have addressed this correctly?

File: java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/window/property/WindowProperty.java [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package org.cloudburstmc.protocol.java.data.window.property;
+
+public interface WindowProperty {
```
Comment (Redned235) [inline java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/window/property/WindowProperty.java]:
```suggestion
public interface ContainerProperty {
```

File: java/java-common/src/main/java/org/cloudburstmc/protocol/java/packet/play/ContainerSetDataPacket.java [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+import org.cloudburstmc.protocol.java.JavaPacket;
+import org.cloudburstmc.protocol.java.data.window.property.WindowProperty;
+import org.cloudburstmc.protocol.java.handler.JavaPlayPacketHandler;
+import org.cloudburstmc.protocol.java.packet.type.JavaPlayPacketType;
+
+@Data
+@EqualsAndHashCode(doNotUseGetters = true, callSuper = false)
+@AllArgsConstructor
+public class ContainerSetDataPacket extends JavaPacket<JavaPlayPacketHandler> {
+    private int windowId;
```
Comment (Redned235) [inline java/java-common/src/main/java/org/cloudburstmc/protocol/java/packet/play/ContainerSetDataPacket.java]:
```suggestion
    private int containerId;
```

File: java/java-common/src/main/java/org/cloudburstmc/protocol/java/packet/play/ContainerSetDataPacket.java [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    private int rawProperty;
```
Comment (Redned235) [inline java/java-common/src/main/java/org/cloudburstmc/protocol/java/packet/play/ContainerSetDataPacket.java]: Should be replaced with a property enum as mentioned on wiki.vg: https://wiki.vg/Protocol#Window_Items

File: java/java-common/src/main/java/org/cloudburstmc/protocol/java/packet/play/ContainerSetSlotPacket.java [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+import org.cloudburstmc.protocol.java.JavaPacket;
+import org.cloudburstmc.protocol.java.data.entity.metadata.ItemStack;
+import org.cloudburstmc.protocol.java.handler.JavaPlayPacketHandler;
+import org.cloudburstmc.protocol.java.packet.type.JavaPacketType;
+import org.cloudburstmc.protocol.java.packet.type.JavaPlayPacketType;
+
+@Data
+@EqualsAndHashCode(doNotUseGetters = true, callSuper = false)
+public class ContainerSetSlotPacket extends JavaPacket<JavaPlayPacketHandler> {
+    private int windowId;
```
Comment (Redned235) [inline java/java-common/src/main/java/org/cloudburstmc/protocol/java/packet/play/ContainerSetSlotPacket.java]:
```suggestion
    private int containerId;
```

**Review by Redned235 — CHANGES_REQUESTED** [review]

**Comment by Redned235** [comment]
Manually merged.

**Pull request merged**